AmplifyPlugins/Internal/Sources/InternalAWSPinpoint/Analytics/ClientError+IsRetryable.swift

```diff
@@ -5,20 +5,19 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-import ClientRuntime
 import Foundation
+import Smithy
+import SmithyHTTPAPI
 
-extension ClientError {
+extension Smithy.ClientError {
     // TODO: Should some of these really be retried?
```

Comment:
Hahaha I guess we haven't :P But let's try to do that.

This is defined on the Analytics context, where we only care if an SDK error is retryable or not when submitting events: If an error is retryable, we will keep the events in the database and try again next time (until they hit the retry count limit). If the error is not retryable, we should remove the events immediately. 

I think this is why pretty much all of the are consider retryable, since they seem to be "transient" errors. The biggest candidate to not being retryable is `.dataNotFound`, but I don't really know when that would be thrown on a `PutEvents` request. 

I guess the answer is then "maybe" 😆